Kentor.AuthServices/SAML2P/Saml2Response.cs

```diff
@@ -473,6 +473,10 @@ private static void CheckSignature(XmlElement signedRootElement, IEnumerable<Asy
         [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Design", "CA1024:UsePropertiesWhereAppropriate")]
         public IEnumerable<ClaimsIdentity> GetClaims(IOptions options)
         {
+            if (options == null)
+            {
+                throw new ArgumentNullException("options");
```

Comment:
I also recall that we recently switched to use the c#6 operator `nameof` elsewhere in the code.
`throw new ArgumentNullException(nameof(options));`

@AndersAbel this might be useful to mention in the contributors guideline?